src/Command/Dev/UpdateComposer/ComposerGenerator.php

```diff
@@ -244,6 +244,9 @@ private function addModules(array $repoOptions, array $composer): array
                     $add($dir);
                 }
             }
+            foreach (glob($baseRepoFolder . '/Inventory*') as $dir) {
```

Comment:
Please add suppress warning

Reply:
Also, we have another package that has the same structure `security-package`. And later we can have other similar packages. 
I think it maybe better to do it more robust.
For example
```
        Inventory:
          branch: 1.0-develop
          repo: https://github.com/magento/Inventory.git
          type: flat-structure
```
And if type == `flat-structure` we will automatically look for modules in the root directory.
@YPyltiai if you agree I can create a task for that and close this PR.

Reply:
Yeah, it sounds good for me, @oshmyheliuk .